base/src/com/google/idea/blaze/base/dependencies/ExternalFileProjectManagementHelper.java

```diff
@@ -124,7 +125,10 @@ public EditorNotificationPanel createNotificationPanel(VirtualFile vf, FileEdito
     }
     boolean inProjectDirectories = AddSourceToProjectHelper.sourceInProjectDirectories(context);
     boolean alreadyBuilt = AddSourceToProjectHelper.sourceCoveredByProjectViewTargets(context);
-    if (alreadyBuilt && inProjectDirectories) {
+    // We do not want to add `/.ijwb` to the project view since it has no BUILD files
+    // This helps to avoid `ERROR: Skipping '//.ijwb/...:all': no targets found beneath '.ijwb'`
+    if (context.file.getPath().contains(String.format("/%s/", BlazeDataStorage.PROJECT_DATA_SUBDIRECTORY))
```

Comment:
We could use `BlazeDataStorage.getAllProjectSubdirectories()` to account for `.clwb` and `.aswb`

Reply:
`  public static final String PROJECT_DATA_SUBDIRECTORY = getProjectDataSubdirectory();` seems already using it and `private static String getProjectDataSubdirectory()` is not visible from the outside.

Reply:
It does, but it will only exclude the one in the IDE you're using. Say someone has some Java and some C++ in the project. They may have to open both with IntelliJ and CLion at the same time, so we have to ignore all of them from every IDE.

I have opened a PR to this PR updating the code to expose an `ALL_PROJECT_SUBDIRECTORIES` constant. Feel free to review it and merge it if useful.